lib/Doctrine/DBAL/Exception/DriverException.php

```diff
@@ -45,13 +45,7 @@ class DriverException extends DBALException
      */
     public function __construct($message, \Doctrine\DBAL\Driver\DriverException $driverException)
     {
-        $exception = null;
-
-        if ($driverException instanceof \Exception) {
```

Comment:
@Ocramius but a bit don't udnerstand if I add interface DriverException extends \Throwable It would still be not possible to pass `parent::__construct($message, 0, $driverException);` because cosntructor expects `\Exception` not `\Throwable`